back/src/routes/messageRouter.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable quotes */
+import express from "express";
+import { messageController } from "../controllers/messageController.js";
+import { catchError } from "../middlewares/catchError.js";
+
+export const messageRouter = new express.Router();
+
+messageRouter.get("/", catchError(messageController.getAll));
```

Comment:
how is this supposed to work if u r only exporting `getAllForRoom` from `messageController` file?

Reply:
Fixed. Thanks. You are so observant)